src/pages/legal/legal.less

```diff
@@ -1,32 +1,34 @@
 #splash {
   .legal {
     display: block !important;
+    margin: 40px auto;
 
```

Comment:
you might want to add `padding: 0 20px` or something for mobile browsers so the text doesn't go right up against the edge